codepropertygraph/src/main/scala/io/shiftleft/passes/CpgPass.scala

```diff
@@ -34,23 +36,28 @@ abstract class CpgPass(cpg: Cpg, outName: String = "", keyPool: Option[KeyPool]
   /**
     * Main method of pass - to be implemented by child class
     * */
-  def run(): Iterator[DiffGraph]
+  def run(): Iterator[DiffGraph] = ???
```

Comment:
Not super happy with this. On the other hand I'd also like to avoid touching hundreds of files for a naming change. Any alternatives that force implementation of either `run` or `runWithExecutionContext`?

Reply:
@hubertp

Reply:
Nope no other way if you don't want to touch those files